python/xrprimer/visualization/keypoints/visualize_keypoints3d.py

```diff
@@ -0,0 +1,364 @@
+# yapf: disable
+from typing import Union
+
+import numpy as np
+
+from xrprimer.data_structure.camera import (
+    FisheyeCameraParameter,
+    PinholeCameraParameter,
+)
+from xrprimer.data_structure.keypoints import Keypoints
+from xrprimer.ops.projection.builder import OpencvProjector
+from xrprimer.transform.convention.world import (
+    BaseWorld,
+    MatplotlibWorld,
+    WorldClass,
+    convert_world,
+)
+from xrprimer.transform.limbs import get_limbs_from_keypoints
+from xrprimer.utils.log_utils import get_logger, logging
+from xrprimer.utils.visualization_utils import rotate_3d_data
+from ..matplotlib.plot_video import plot_video
+from ..palette import LinePalette, PointPalette, get_different_colors
+from ..presets import create_coordinate_axis
+from .visualize_keypoints2d import visualize_keypoints2d
+
+# yapf: enable
+
+
+def visualize_keypoints3d_cv2(
+    # input args
+    keypoints: Keypoints,
+    camera: Union[PinholeCameraParameter, FisheyeCameraParameter],
```

Comment:
What about setting a default value for camera? User can call this function more easily in this way.

Reply:
Is there more appropriate name? like `visualize_projected_keypoints3d`?

Reply:
> What about setting a default value for camera? User can call this function more easily in this way.

If user's keypoints are not near the origin he will get empty results. Also the person object in keypoints may be plotted upside down if it does not match our default camera. 
I think the compatibility of the default camera is poor and its effective utilization rate is low, thus it is not provided. In addition, most publicly available datasets also provide their own cameras.

Reply:
> Is there more appropriate name? like `visualize_projected_keypoints3d`?
Any 3D object we see on a 2D screen is a projection under a certain viewpoint, so I think it is inappropriate to name this function solely as "projection". Matplotlib, and babylon in the future, also do projection. The main difference that can be determined is that their implementations use different third-party libraries.

Reply:
According to discussion offline, `visualize_keypoints3d_plt` will be renamed to `visualize_keypoints3d`, as a naive method for keypoints3d visualization. `visualize_keypoints3d_cv2` will be renamed to `visualize_keypoints3d_projected`, more complicated than the naive one.

Reply:
Fixed in [b0767c4](https://github.com/openxrlab/xrprimer/pull/39/commits/b0767c45c9f4e3c349a6566e93145f23c3095e7c).